pkgs/applications/audio/ams-lv2/default.nix

```diff
@@ -12,13 +12,13 @@ stdenv.mkDerivation  rec {
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ cairo fftw gtkmm2 lv2 lvtk python ];
+  buildInputs = [ cairo fftw gtkmm2 lv2 lvtk ];
 
-  configurePhase = "python waf configure --prefix=$out";
+  configurePhase = "${python3.interpreter} waf configure --prefix=$out";
```

Comment:
Why not just change `python -> python3`? These lines looked better before.

Reply:
Because absolute paths are always better.

Reply:
Also, for example, someone might want to replace `python3` by a different implementation like `pypy`, which does not provide the `python` executable afaik.

Reply:
Ah alright, the second reason is one I can live with :)